app/src/main/kotlin/com/saltedge/authenticator/features/connections/delete/DeleteConnectionDialog.kt

```diff
@@ -26,21 +26,24 @@ import android.os.Bundle
 import androidx.appcompat.app.AlertDialog
 import androidx.fragment.app.DialogFragment
 import androidx.fragment.app.activityViewModels
+import androidx.navigation.fragment.navArgs
 import com.saltedge.authenticator.R
 import com.saltedge.authenticator.features.main.SharedViewModel
 import com.saltedge.authenticator.sdk.model.GUID
-import com.saltedge.authenticator.tools.guid
 
 class DeleteConnectionDialog : DialogFragment(),
     DeleteConnectionContract.View,
     DialogInterface.OnClickListener
 {
     private var presenter = DeleteConnectionPresenter(viewContract = this)
     private val sharedViewModel: SharedViewModel by activityViewModels()
+    private val safeArgs: DeleteConnectionDialogArgs by navArgs()
+    private val guid: String
```

Comment:
`safeArgs` and `guid` redundant values. 
try `presenter.guid = navArgs().guid`

Reply:
Your solution doesn't work